<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>directManageWidget</class>
 <widget class="QWidget" name="directManageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>371</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="refPosButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>В референсную</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disSteppersButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отключить ШД</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Текущая позиция</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Х = </string>
    </property>
   </widget>
   <widget class="QLabel" name="posXLable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y = </string>
    </property>
   </widget>
   <widget class="QLabel" name="posYLable">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="posZLable">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="GPushButton" name="zeroAllButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Обнулить все</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z = </string>
    </property>
   </widget>
   <widget class="GPushButton" name="zeroXButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Обнулить</string>
    </property>
   </widget>
   <widget class="GPushButton" name="zeroYButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Обнулить</string>
    </property>
   </widget>
   <widget class="GPushButton" name="zeroZButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Обнулить</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Скорость перемещения</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>мм/с</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="speedSpinBox">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <double>999.990000000000009</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>F = </string>
    </property>
   </widget>
   <widget class="QLabel" name="speedLable">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>G Код</string>
   </property>
   <widget class="QLineEdit" name="GlineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendGCode">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Отработать</string>
    </property>
   </widget>
  </widget>
  <widget class="GPushButton" name="Yp10Button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>310</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Y+10</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Yp1Button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Y+1</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Yp01Button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Y+0.1</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Ym01Button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-0.1</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Ym10Button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-10</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Ym1Button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-1</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Xp01Button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>160</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>X+0.1</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Xp1Button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>160</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>X+1</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Xp10Button">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>160</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>X+10</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Xm1Button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>X-1</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Xm10Button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>X-10</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Xm01Button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>160</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>X-0.1</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Zm1Button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Z-1</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Zm10Button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Z-10</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Zp10Button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>310</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Z+10</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Zp01Button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>210</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Z+0.1</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Zp1Button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>260</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Z+1</string>
   </property>
  </widget>
  <widget class="GPushButton" name="Zm01Button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>110</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Z-0.1</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Концевики</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ось Х - </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ось Y -</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ось Z -</string>
    </property>
   </widget>
   <widget class="QLabel" name="swXLable">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Разомкнут</string>
    </property>
   </widget>
   <widget class="QLabel" name="swYLable">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Разомкнут</string>
    </property>
   </widget>
   <widget class="QLabel" name="swZLable">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Разомкнут</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="zeroPosButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>В нулевую</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Стоп</string>
   </property>
   <property name="icon">
    <iconset theme="1">
     <normaloff>../STOP.jpg</normaloff>../STOP.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>81</width>
     <height>81</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Ограничение рабочей зоны</string>
   </property>
   <widget class="QLabel" name="endPosYLable">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Х = </string>
    </property>
   </widget>
   <widget class="QLabel" name="endPosXLable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Z = </string>
    </property>
   </widget>
   <widget class="QLabel" name="endPosZLable">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y = </string>
    </property>
   </widget>
   <widget class="GPushButton" name="setEndPosButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ограничить рабочую зону&lt;/p&gt;&lt;p&gt;текущими координатами.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Ограничить</string>
    </property>
   </widget>
   <widget class="GPushButton" name="drawWorkZoneButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ограничить рабочую зону&lt;/p&gt;&lt;p&gt;текущими координатами.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Расчертить</string>
    </property>
   </widget>
   <widget class="GPushButton" name="resetWorkZoneButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ограничить рабочую зону&lt;/p&gt;&lt;p&gt;текущими координатами.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Сброс</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">widgets/gpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
